pDESy/model/base_priority_rule.py

```diff
@@ -82,5 +93,10 @@ def sort_task_list(task_list, priority_rule_mode=TaskPriorityRuleMode.TSLACK):
             task_list = sorted(
                 task_list, key=lambda task: task.default_work_amount, reverse=True
             )
+        elif priority_rule_mode == TaskPriorityRuleMode.FIFO:
+            # Task: FIFO (First In First Out rule)
+            task_list = sorted(
+                task_list, key=lambda task: task.state_record_list.count(1),reverse=True
```

Comment:
Please use `BaseTaskState.READY`, not `1` or other numerical.

Reply:
> 
```suggestion
                task_list, key=lambda task: task.state_record_list.count(BaseTaskState.READY),reverse=True
```

error code 
`AILED tests/model/test_base_priority_rule.py::test_sort_task_list_FIFO - NameError: name 'BaseTaskState' is not defined`